clients/drcachesim/common/trace_entry.h

```diff
@@ -612,6 +612,13 @@ typedef enum {
      */
     TRACE_MARKER_TYPE_CONTEXT_SWITCH_END,
 
+    /**
+     * The marker is used to indicate the current vector length in bytes for
```

Comment:
When we start adding this marker on `prctl(PR_SVE_SET_VL, ..)` calls, we might need to somehow add the marker in all thread's buffers the moment the change occurred. (I assume the prctl call changes the vector size for all threads.) Add a TODO?

Reply:
> (I assume the prctl call changes the vector size for all threads.)

`prctl(PR_SVE_SET_VL, ..)` only changes the vector length for the current thread so that shouldn't necessary.
https://man7.org/linux/man-pages/man2/prctl.2.html (search for PR_SVE_SET_VL)